<launch>

  <arg name="model" default="$(find wpm2_bringup)/urdf/wpm2_moveit.urdf"/>
  <arg name="gui" default="true" />
  <arg name="rvizconfig" default="$(find wpm2_bringup)/rviz/urdf.rviz" />

  <param name="robot_description" command="$(find xacro)/xacro $(arg model)" />
  <param name="use_gui" value="$(arg gui)"/>

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />

  <node name="wpm2_core" pkg="wpm2_bringup" type="wpm2_core" output="screen"/>

  <node name="wpm2_trackbars" pkg="wpm2_tutorials" type="wpm2_trackbars" output="screen"/>

</launch>
